<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Adenine</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="adenineChangeColorButton">
         <property name="text">
          <string>Change color...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Cytosine</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cytosineChangeColorButton">
         <property name="text">
          <string>Change color...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Guanine</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="guanineChangeColorButton">
         <property name="text">
          <string>Change color...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Thymine</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="thymineChangeColorButton">
         <property name="text">
          <string>Change color...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
